command/harness/dlite/init.go

```diff
@@ -0,0 +1,74 @@
+package dlite
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/drone-runners/drone-runner-aws/command/harness"
+	"github.com/wings-software/dlite/client"
+	"github.com/wings-software/dlite/logger"
+)
+
+type VMInitTask struct {
+	c *dliteCommand
+}
+
+type VMInitRequest struct {
+	SetupVMRequest harness.SetupVMRequest      `json:"setup_vm_request"`
+	Services       []*harness.ExecuteVMRequest `json:"services"`
+}
+
+func (t *VMInitTask) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background() // TODO: Get this from the request
+	task := &client.Task{}
+	err := json.NewDecoder(r.Body).Decode(task)
+	if err != nil {
+		logger.WriteBadRequest(w, err)
+		return
+	}
+	// Unmarshal the task data
+	taskBytes, err := task.Data.MarshalJSON()
+	if err != nil {
+		logger.WriteBadRequest(w, err)
+		return
+	}
+	req := &VMInitRequest{}
+	err = json.Unmarshal(taskBytes, req)
+	if err != nil {
+		logger.WriteBadRequest(w, err)
```

Comment:
logger.WriteBadRequest

The package name for it needs to be changed. I thought it was trying to write something to the logs